cartridges/int_adyen_overlay/cartridge/scripts/adyenCheckout.ds

```diff
@@ -232,6 +232,24 @@ function doPaymentCall(args, requestObject){
                 Logger.getLogger('Adyen').debug('Adyen: ' + resultObj.statusCode + ' Error => ' + resultObj.statusMessage + ' | ' + resultObj.errorText);
             }
         }
+        else if(responseObj.resultCode.indexOf('PresentToShopper') != -1) {
```

Comment:
I found it several times that we are checking if the string starts with an expected string but why don't we just check if the string is the expected string or not?
```suggestion
        else if(responseObj.resultCode == 'PresentToShopper') {
```

Reply:
Good one, I will update it